input/params/simulations/Genesis_L210_N4320.par

```diff
@@ -2,9 +2,9 @@
 #----- Simulation input files  ------------
 #------------------------------------------
 SimName           : Genesis L210_N4320
-SimulationDir     : /fred/oz113/balu/files_for_meraxes/L210_N4320/un_augmented_trees
+SimulationDir     : /fred/oz113/yqin/dragons/input_trees/Genesis_L210_N4320
```

Comment:
you can take a look at this directory. there, i stored both @s-balu 's original trees and @EMventura 's trimmed augmented trees.

Reply:
Shouldn't it be /fred/oz113/yqin/input_trees/Genesis_L210_N4320_aug_trimmed ?

Reply:
no, see what's inside the directory and `read_halos-velociraptor.c`

Reply:
Yeah I got it, you changed the names! It makes sense.